OWML.ModHelper.Menus/ModComboInput.cs

```diff
@@ -0,0 +1,139 @@
+﻿using OWML.Common.Menus;
+using OWML.ModHelper.Events;
+using OWML.Common;
+using UnityEngine;
+using UnityEngine.UI;
+using System;
+
+namespace OWML.ModHelper.Menus
+{
+    public class ModComboInput : ModInput<string>, IModComboInput
+    {
+
+        private string _value;
+        private Vector3 scale;
+        private UIStyleManager styleManager;
+        private HorizontalLayoutGroup _layoutGroup;
+        public IModLayoutButton Button { get; }
+        protected readonly IModInputMenu InputMenu;
+        protected readonly TwoButtonToggleElement ToggleElement;
```

Comment:
public/protected variables should be above private variables, and you can separate them with a newline

Reply:
done